cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/00/Readme.txt

```diff
@@ -0,0 +1,23 @@
+--The database needs to be renamed.
+
+DEV:	pituwdev
+QA:		pituwqa
+DLVR:	pituwdlvr
+TEST:	pituwtest
+PROD:	pituwprod
+
+--Rename sql has to be run: ALTER DATABASE old_database_name RENAME TO new_database_name;
+
+DEV:	ALTER DATABASE cirrasdev RENAME TO pituwdev;
+QA:		ALTER DATABASE cirrasqa RENAME TO pituwqa;
+DLVR:	ALTER DATABASE cirrasdlvr RENAME TO pituwdlvr;
+TEST:	ALTER DATABASE cirrastest RENAME TO pituwtest;
+PROD:	ALTER DATABASE cirrasprod RENAME TO pituwprod;
+
+
+Possible Error if there are sessions using the database:
+	There are X other sessions using the database.database "cirras_dev" is being accessed by other users 
+	ERROR:  database "cirrasdev" is being accessed by other users
+	SQL state: 55006
+	
+Solution: SELECT pg_terminate_backend( pid ) FROM pg_stat_activity WHERE pid <> pg_backend_pid( ) AND datname = 'cirrasdev'; //WHERE cirrasdev is the old database name
```

Comment:
I would recommend we shutdown the UW API prior to renaming the database. That should hopefully either reduce or eliminate the number of connections that have to be manually terminated.

Reply:
Also shutdown Jasper; it will still be running in some environments.